Makefile.am

```diff
@@ -24,7 +24,6 @@ ACLOCAL_AMFLAGS = -I aclocal
 install-data-hook:
 	if [ ! -d $(DESTDIR)$(statedir) ]; then mkdir -p $(DESTDIR)$(statedir); fi
 	touch $(DESTDIR)$(statedir)/xtab; chmod 644 $(DESTDIR)$(statedir)/xtab
```

Comment:
What is `xtab` used for, and do we need to also remove this line?

Reply:
`xtab` is leftover from ancient kernels which didn't have `/proc/fs/nfs/export`. It is removed in the latest upstream nfs-utils.  We don't need to remove it here since it is unused in our system.